blaze/server/tests/test_server.py

```diff
@@ -583,72 +518,75 @@ def test_isin(test, serial):
 
 
 @pytest.mark.parametrize('serial', all_formats)
-def test_add_data_to_empty_server(empty_server, serial):
-    # add data
-    with temp_server() as test:
+def test_add_data_twice_error(empty_server, serial):
+    with temp_server():
+        # add iris
         iris_path = example('iris.csv')
-        blob = serial.dumps({'iris': iris_path})
-        response1 = empty_server.post(
-            '/add',
-            headers=mimetype(serial),
-            data=blob,
-        )
-        assert 'OK' in response1.status
-        assert response1.status_code == 200
+        payload = serial.dumps({'iris': iris_path})
+        empty_server.post('/add',
+                          headers=mimetype(serial),
+                          data=payload)
+        # Try to add to existing 'iris'
+        resp = empty_server.post('/add',
+                                 headers=mimetype(serial),
+                                 data=payload)
+        assert resp.status_code == 409 # CONFLICT
```

Comment:
are these error codes not defined somewhere, and if not, should we make a small namespace for them like: `rc.CONFLICT`